Neos.Neos/Classes/Domain/Service/FusionSourceCodeFactory.php

```diff
@@ -50,19 +53,12 @@ public function createFromAutoIncludes(): FusionSourceCodeCollection
         $sourcecode = FusionSourceCodeCollection::empty();
         foreach (array_keys($this->packageManager->getAvailablePackages()) as $packageKey) {
             if (isset($this->autoIncludeConfiguration[$packageKey]) && $this->autoIncludeConfiguration[$packageKey] === true) {
-                $sourcecode = $sourcecode->union(
-                    FusionSourceCodeCollection::tryFromPackageRootFusion($packageKey)
-                );
+                $sourcecode = $this->fusionAutoIncludeHandler->loadFusionFromPackage($packageKey, $sourcecode);
             }
         }
         return $sourcecode;
     }
 
-    public function createFromSite(Site $site): FusionSourceCodeCollection
-    {
-        return FusionSourceCodeCollection::tryFromPackageRootFusion($site->getSiteResourcesPackageKey());
-    }
-
```

Comment:
this method was removed as its considered internal and relatively new code from myself :D

It was a yagni thing and we didn't even use it once :D

Reply:
@kitsunet if you would prefer rather a deprecation here i would also be fine with that ...

Reply:
def deprecation yeah, see also other comment above.

Reply:
👍 did it :)